extension/src/test/suite/setup/index.test.ts

```diff
@@ -579,7 +578,6 @@ suite('Setup Test Suite', () => {
       mockRunSetup.restore()
       stub(config, 'isPythonExtensionUsed').returns(false)
       stub(config, 'getPythonBinPath').resolves(join('python'))
-      stub(setup, 'getDvcCliDetails').resolves(undefined)
```

Comment:
With `getDvcCliDetails` no longer calling `getDvcVersion`, we can delete related stubs.